### ISSUE TITLE
Endless loading state on "Modern Properties" tab if App Catalog excluded from search results

### ISSUE THREAD

**Issue by IvanTheBearable** [tavikukko/Chrome-SP-Editor#69]
Lines 2297 - 2299 of /app/panelspqueries.js perform a search for the app catalog site. If no app catalog site is found (because it hasn't been created or it has been excluded from search results) then this search returns zero results. There is no check for this scenario. Line 2300 attempts to read the first result and fails with the following error:

```
Uncaught (in promise) TypeError: Cannot read property 'SiteId' of undefined
    at <anonymous>:178:51
```

**Comment by tavikukko** [comment]
Thanks @IvanTheBearable ,
I will have a look.

**Comment by tavikukko** [comment]
Thanks @IvanTheBearable for the PR!
I have merged it and fix will be available in version 5.5.4 when Google approves the updated package.

**Comment by tavikukko** [comment]
Updated version 5.5.4 released and available
